modules/swagger-codegen/src/main/resources/ruby/partial_model_generic.mustache

```diff
@@ -36,18 +36,18 @@
     # Attribute mapping from ruby-style variable name to JSON key.
     def self.attribute_map
       {
-        {{#vars}}
+        {{#allVars}}
         :'{{{name}}}' => :'{{{baseName}}}'{{#hasMore}},{{/hasMore}}
-        {{/vars}}
+        {{/allVars}}
       }
     end
 
     # Attribute type mapping.
     def self.swagger_types
       {
-        {{#vars}}
+        {{#allVars}}
         :'{{{name}}}' => :'{{{datatype}}}'{{#hasMore}},{{/hasMore}}
-        {{/vars}}
+        {{/allVars}}
```

Comment:
Due to the way the deserializer works in the api_client, the parent values must be present in `swagger_types` and `attribute_map`